### PR TITLE
Create a .dokerignore file to minimize Docker image

### PR DESCRIPTION
**Pull request by Pea-skillz**
Signed-off-by: Pierre-Emmanuel Andre <pierre-emmanuel.andre@skillz.io>

* **Analysis**: It's not necessary to put all the files in the Docker image

* **Solution**: Create a .dockerignore file

* **Performed tests**: Run TRD

* **Issue**: [Issue 546](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/issues/546)

**Work effort**: 1.5h

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# hidden files and directories
+.*
+.*/
```
Comment (vkresch) [inline .dockerignore:3]: Are we sure we want to always ignore hidden files and folders? 
I would prefer rather using explicit folder and file names and keep the option.

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# files
+tezos-reward.service_template
+service_add.py
+*.csv
```
Comment (vkresch) [inline .dockerignore:8]: We can also ignore all *.md files like the readme.md.

**Review by vkresch — CHANGES_REQUESTED** [review]
Looks good thanks for the PR. We might add also .md extension to the ignore file and be explicit regarding the hidden files and folders. Also we can ignore __pycache__ folders and pyc files. Please get some inspiration from the .gitignore file.

**Comment by jdsika** [comment]
@Pea-skillz please adhust to the comments. I did send an invite to the TRD organisation and support team. Then you can create branches instead of forks for reviewers to make small changes themselves.

File: .dockerignore (hunk 2 above)
Comment (Pea-skillz) [reply .dockerignore:15]: OK. Fixed

File: .dockerignore (hunk 1 above)
Comment (Pea-skillz) [reply .dockerignore]: Ok, looks good to me :)
Fixed in my last commit

File: .dockerignore (hunk 2 above)
Comment (Pea-skillz) [reply .dockerignore:15]: Fixed too

**Comment by Pea-skillz** [comment]
@jdsika ok noted. Will do for my next PR :)

**Review by jdsika — APPROVED** [review]

**Review by vkresch — APPROVED** [review]

**Pull request merged**